content/exsaml.xqm

```diff
@@ -87,8 +87,8 @@ declare function exsaml:info() {
  : @param relaystate this is the path component of the resource that the user
  :  initially requested, so that she gets sent there after SAML auth.
  :)
-declare function exsaml:build-authnreq-redir-url($relaystate as xs:string) {
-    let $log := exsaml:log("info", "building SAML auth request redir-url; relaystate: " || $relaystate)
+declare function exsaml:build-authnreq-redir-url($relayurl as xs:string, $realm as xs:string) {
```

Comment:
This is a breaking change for any app that relies on `exsaml:build-authnreq-redir-url#1` (the original arity-1 version of this function). For backwards compatibility, would it be possible to retain the arity-1 version of this function? If that's not possible, this PR would to be released in a new major version of the package.

Reply:
@joewiz I was planning to do exactly that, for backwards compatibility

Reply:
Great! Hopefully that will allow my existing apps that use existdb-saml (e.g., hsa, guides, hma) to continue working without modification. Of course, if there's an important breaking change, we can always change the other apps.